### PR TITLE
ci: adds workflow to deploy to Azure SWA

### PR DESCRIPTION
**Pull request by Ensar025**

### CODE REVIEW

File: .github/workflows/verify.yml [hunk 1]
```diff
@@ -42,4 +42,65 @@ jobs:
...
+      - name: Copy package-lock from root
+        run: cp package-lock.json packages/backend
+
+      - name: Copy node_modules from root
+        run: cp -R node_modules packages/backend
+
+      - name: Remove @dependency-health-monitor from node_modules in backend
+        run: cd packages/backend && rm -rf node_modules/@dependency-health-monitor
+
+      - name: Install SWA
```
Comment (vanhoofmaarten) [inline .github/workflows/verify.yml:95]: It's not considered best practice to install globally on CI/CD. This is why the `npx` command was invented. See next comment.

File: .github/workflows/verify.yml [hunk 2]
```diff
@@ -42,4 +42,65 @@ jobs:
...
+
+      - name: Deploy Azure Static Web App
+        env:
+          DEPLOYMENT_TOKEN: ${{ secrets.SWA_CLI_DEPLOYMENT_TOKEN }}
+          SUBSCRIPTION_ID: ${{ secrets.AZURE_SUBSCRIPTION_ID }}
+          TENANT_ID: ${{ secrets.AZURE_TENANT_ID }}
+          APP_NAME: ${{ secrets.SWA_CLI_APP_NAME }}
+          RESOURCE_GROUP: ${{ secrets.AZURE_RESOURCE_GROUP }}
+
+        run: swa deploy -a ./packages/frontend -i ./packages/backend -O ./dist -d $DEPLOYMENT_TOKEN -S $SUBSCRIPTION_ID -T $TENANT_ID -n $APP_NAME -R $RESOURCE_GROUP
```
Comment (vanhoofmaarten) [inline .github/workflows/verify.yml:106]: As per my previous comment, here you can use `npx @azure/static-web-apps-cli` in favour of the globally installed package.

**Review by vanhoofmaarten — CHANGES_REQUESTED** [review]

File: .github/workflows/verify.yml (hunk 1 above)
Comment (Ensar025) [reply .github/workflows/verify.yml]: Resolved in fcb783d9b104aaaa75a840e823d2e1188be11108

File: .github/workflows/verify.yml (hunk 2 above)
Comment (Ensar025) [reply .github/workflows/verify.yml]: Resolved in fcb783d9b104aaaa75a840e823d2e1188be11108

**Review by vanhoofmaarten — APPROVED** [review]

**Pull request merged**